messages/data.bulk.results.md

```diff
@@ -1,22 +1,22 @@
 # summary
 
-Get the results of a Bulk API 2.0 ingest job.
+Get the results of a bulk ingest command that you previously ran.
 
 # description
 
-You can use this command to get successful/failed/unprocessed results of a bulk 2.0 ingest job written to different files in comma-separated values (CSV) format.
+Use this command to get the complete results after running one of the CLI commands that uses Bulk API 2.0 to ingest (import, upsert, or delete) large datasets to your org, such as "data import bulk". The previously-run bulk command must have completed; if it's still processing, run the corresponding resume command first, such as "data import resume." Make note of the job ID of the previous bulk command because you use it to run this command.
 
-If the job was aborted/failed you might also get unprocessed results.
+This command first displays the status of the previous bulk command, the operation that was executed in the org (such as insert or hard delete), and the updated Salesforce object. The command then displays how many records were processed in total, and how many were successful or failed. Finally, the output displays the names of the generated CSV-formatted files that contain the specific results for each ingested record. Depending on the success or failure of the bulk command, the results files can include the IDs of inserted records or the specific errors. When possible, if the ingest job failed or was aborted, you also get a CSV file with the unprocessed results.
 
 # flags.job-id.summary
 
 Job ID of the bulk job.
 
 # examples
 
-- Get results from a bulk import job. Use the org with alias "my-scratch"
+- Get results from a bulk import command; use the org with alias "my-scratch":
```

Comment:
This command isn't limited to bulk jobs started by other `sf` commands, you can pass any valid job ID that was created by other tool like the [sfdmu plugin](https://github.com/forcedotcom/SFDX-Data-Move-Utility) or [data loader](https://developer.salesforce.com/docs/atlas.en-us.dataLoader.meta/dataLoader/loader_configuring_bulk_api_20.htm?q=bulk).

Reply:
Oh!  I didn't know that, let me add a note about it to the help.  I think that's a pretty cool feature...

Reply:
@cristiand391 , I updated the help to mention using the command to see results from jobs in general,  and I also mostly used "job" rather than "command", now that it's more generic.  You had originally done that, and now I see why!  I'll resolve these two conversations for now, but please let me know if there are other changes.